src/Organic/Organic.php

```diff
@@ -1071,6 +1074,28 @@ public function fullResyncContent( $batch = 50, $offset = 0, $sleep_between = 0
         return $updated;
     }
 
+    public function getContentResyncStartedAt(): DateTimeImmutable {
+        return DateTimeImmutable::createFromFormat(
+            DATE_ATOM,
+            $this->getOption(
+                'organic::content_resync_started_at',
+                '2003-05-27T00:00:00+00:00'
```

Comment:
Just a fun Easter Egg since there needed to be a default date sometime in the past. https://wordpress.org/news/2003/05/wordpress-now-available/

Reply:
I was going to ask about this particular default :)

Reply:
Updated to include the appropriate time. Accurate data is very important.